citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/AuthorityItemsService.java

```diff
@@ -0,0 +1,22 @@
+package edu.asu.diging.citesphere.core.authority.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.citesphere.core.authority.IAuthorityItemsService;
+import edu.asu.diging.citesphere.data.bib.ICitationDao;
+import edu.asu.diging.citesphere.model.authority.IAuthorityEntry;
+import edu.asu.diging.citesphere.model.transfer.impl.Citations;
+
+@Service
+public class AuthorityItemsService implements IAuthorityItemsService {
```

Comment:
Is there a reason this is not part of the citation manager class? Also, tests are missing.